src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -255,13 +255,16 @@
                   min="0"
                   class="p-2"
                   placeholder="ex: 50"
-                  :class="{ 'is-danger': errors.transfer.balance }"
+                  :class="{
+                    'is-danger':
+                      errors.transfer.balance || errors.transfer.amount,
+                  }"
                 />
                 <div
                   class="notification is-danger is-light"
-                  v-if="errors.transfer.balance"
+                  v-if="errors.transfer.balance || errors.transfer.amount"
                 >
-                  {{ errors.transfer.balance }}
+                  {{ errors.transfer.balance || errors.transfer.amount }}
```

Comment:
You are sure you want to display only the first one if both are set ?

Reply:
Since `errors.transfer.amount` will only be triggered when amount is negative, the first one will never be set if the second one is.

Reply:
@vaab is that ok with you ?